README.md

```diff
@@ -13,7 +13,7 @@
 ## Directory Structure
 
 <pre>
-├── <a href="./bridge-history-api/">bridge-history-api</a>: Bridge history service that collects deposit and withdraw events from both L1 and L2 chain and generates withdrawal proofs
+├── <a href="./bridge-history-api/">bridge-history-api</a>: Bridge history service that collects deposit and withdraws events from both L1 and L2 chains and generates withdrawal proofs
```

Comment:
Should be "deposit and withdraw events"

Reply:
that is true, I modified those, thank you for your feedback !